_includes/header.html

```diff
@@ -1,6 +1,6 @@
 <header class="site-header">
   <div class="container">
-    <h1><a href="{{ '/' | relative_url }}">Mod dotnet-bot</a></h1>
+    <a style="color:white;font-weight:600;font-size:20px;margin-right:48px" href="{{ '/' | relative_url }}">Mod dotnet-bot</a>
```

Comment:
@danzhu54 can you check if you can move this to the proper file? I tried but it didn't seem to work so added here manually.

Reply:
We should probably move this to https://github.com/dotnet-foundation/mod-dotnet-bot/blob/32808d87b4cb76200622753715d657bd9e7c92af/_dev/scss/layout/_header.scss. I can add it.

Reply:
I think that's where I tried and it didn't work. But maybe I did something wrong 🤷🏻‍♀️

Reply:
The gulp task is supposed to compile the sass into main.css under ./assets/css but I'm running into a compilation issue at the moment. We we can merge these changes for now.